kuma/landing/jinja2/landing/fellowship.html

```diff
@@ -1,197 +0,0 @@
-{% extends "wiki/base.html" %}
```

Comment:
There are still references to this in the `.po` files. I am guessing this will be removed as part of future localisation updates?